resources/profiles/Qidi/filament/Qidi Generic ABS @Qidi XMax3 0.2 nozzle.json

```diff
@@ -0,0 +1,32 @@
+{
+    "type": "filament",
+    "filament_id": "GFB99",
+    "setting_id": "GFSA04",
+    "name": "Qidi Generic ABS @Qidi XMax3 0.2 nozzle",
+    "from": "system",
+    "instantiation": "true",
+    "inherits": "fdm_filament_abs",
+    "filament_flow_ratio": [
+        "0.926"
+    ],
+    "enable_pressure_advance": [
+        "1"
+    ],
+    "pressure_advance": [
+        "0.021"
+    ],
+    "filament_max_volumetric_speed": [
+        "2"
+    ],
+    "filament_start_gcode": [
+        "; filament start gcode\n{if  (bed_temperature[current_extruder] >90)||(bed_temperature_initial_layer[current_extruder] >90)}M106 P3 S0\n{elsif(bed_temperature[current_extruder] >70)||(bed_temperature_initial_layer[current_extruder] >70)}M106 P3 S153\n{elsif(bed_temperature[current_extruder] >35)||(bed_temperature_initial_layer[current_extruder] >35)}M106 P3 S255\n{endif}"
+    ],
+	"compatible_printers": [
+        "Qidi X-Plus 0.2 nozzle",
+        "Qidi X-Max 0.2 nozzle",
+        "Qidi X-CF Pro 0.2 nozzle",
+		"Qidi X-Smart 3 0.2 nozzle",
+		"Qidi X-Plus 3 0.2 nozzle",
+		"Qidi X-Max 3 0.2 nozzle"
```

Comment:
Since this profile is specialized for `Qidi X-Max 3 0.2 nozzle`, we should remove all other printers